.github/gbp.toml

```diff
@@ -9,6 +9,7 @@ reset_label = "GBP: Reset"
 "Feedback" = 2
 "Fix" = 3
 "Grammar and Formatting" = 1
+"Hard Deletes" = 5
```

Comment:
Performance is 12, but Hard Deletes, the major source of perf improvements, is only 5?
Outrageous.

Reply:
performance is 12? damn
was just kinda a ballpark number honestly

Reply:
Perf good. Resolving hard dels good. Give it GBPs, we really want this kind of PR.

Reply:
performance hsould just come with hard deletes

Reply:
You think? Part of me hopes that seeing a fancy label will encourage people to look into them

Reply:
i want them both

Reply:
but any pr with hard deletes should also have performance is what i mean

Reply:
want me to make it a bit less valuable then? or should we just shower em in it

Reply:
> 
> 
> but any pr with hard deletes should also have performance is what i mean

Performance is for prs with information about a specific performance increase and change meant specifically to improve performance. A plain hard del removal is a code cleanliness and stability thing that doesn't matter for performance most of the time.

Now if you fix a particularly big one that's different.